### PR TITLE
update scripts

### PR DESCRIPTION
**Pull request by rcannood**
## Checklist before requesting a review
- [ ] I have performed a self-review of my code

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [ ] New functionality
  - [ ] Major changes
  - [ ] Minor changes
  - [ ] Bug fixes

- [ ] Proposed changes are described in the CHANGELOG.md

- [ ] CI Tests succeed and look good!

### CODE REVIEW

File: scripts/project/build_all_components.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+# Build all components in a namespace (refer https://viash.io/reference/cli/ns_build.html)
+viash ns test --parallel
```
Comment (KaiWaldrant) [inline scripts/project/build_all_components.sh:6]: the file name says build but the  cmd is for test? not sure if `test` is wrong or the filename is wrong

File: scripts/project/build_all_docker_containers.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+# Build all components in a namespace (refer https://viash.io/reference/cli/ns_build.html)
+# and set up the container via a cached build
+viash ns test --parallel --setup cachedbuild
```
Comment (KaiWaldrant) [inline scripts/project/build_all_docker_containers.sh:7]: same here

**Review by KaiWaldrant — CHANGES_REQUESTED** [review]
Minor changes

File: scripts/project/build_all_components.sh (hunk 1 above)
Comment (rcannood) [reply scripts/project/build_all_components.sh]: Thanks for noticing!

File: scripts/project/build_all_docker_containers.sh (hunk 2 above)
Comment (rcannood) [reply scripts/project/build_all_docker_containers.sh]: Same here!

**Review by KaiWaldrant — APPROVED** [review]
LGTM!

**Pull request merged**